src/block_cache/block_chunkcache.c

```diff
@@ -916,6 +916,52 @@ __wt_chunkcache_get(WT_SESSION_IMPL *session, WT_BLOCK *block, uint32_t objectid
     return (0);
 }
 
+/*
+ * __wt_chunkcache_free_external --
+ *     Find chunks in the chunk cache using object id, and free the chunks.
+ */
+int
+__wt_chunkcache_free_external(
+  WT_SESSION_IMPL *session, WT_BLOCK *block, uint32_t objectid, wt_off_t offset, uint32_t size)
+{
+    WT_CHUNKCACHE *chunkcache;
+    WT_CHUNKCACHE_CHUNK *chunk, *chunk_tmp;
+    WT_CHUNKCACHE_HASHID hash_id;
+    size_t already_read;
+    uint64_t bucket_id;
+    const char *object_name;
+    chunkcache = &S2C(session)->chunkcache;
```

Comment:
For readability - could you split the declarations and assignments into another paragraph please?

Reply:
this was my bad - of course. [6d1b1da](https://github.com/wiredtiger/wiredtiger/pull/9838/commits/6d1b1da34e6d1213840588e9088bfc8f1c674881)